CMakeLists.txt

```diff
@@ -148,12 +148,12 @@ find_library(FOUND_UNWIND_LIB unwind)
 set(WARNING_FLAGS -Wall -Wpedantic -Wsign-compare -Wreturn-type)
 
 # kvrocks objects target
-file(GLOB KVROCKS_SRCS src/*.cc)
+file(GLOB_RECURSE KVROCKS_SRCS src/*.cc)
```

Comment:
Does the build benefit from separating compile units? For example, one `.o` per directory and for better incremental compilation. If it does we can implement it here or as a follow-up.